app/models/review.rb

```diff
@@ -0,0 +1,7 @@
+class Review < ApplicationRecord
+  
+  validates :rating, numericality: {less_than_or_equal_to: 5} 
+  validates :comment, length: {maximum: 500}, allow_blank: true
```

Comment:
these look good, please add a space between the curly brackets and the key value pairs in your hashes